tor.rst

```diff
@@ -0,0 +1,178 @@
+Using Electrum Through Tor
+==========================================================
+
+Please note that when using electrum through tor there is two main ways.
+The first way has the most Privacy but also requires the most trust in the server you are connecting too. This is because normally
+Electrum connects to a few different servers and downloads block headers and checks that they match. This prevents / makes it more difficult for 
+Rogue servers to send you bad information. However this can also present a privacy issue because you could be connecting to none .onion servers for these headers.
```

Comment:
However this can also present a privacy issue because you could be connecting to none .onion servers for these headers.

You would still connect through the Tor network so your privacy shouldn't be affected.